src/test/resources/contracts/agency-locations.avsc

```diff
@@ -0,0 +1,294 @@
+{
+  "type": "record",
+  "namespace": "prisons",
+  "service": "nomis",
+  "version": "1.0.0",
+  "name": "AGENCY_LOCATIONS",
+  "description": "A physical location of an agency involved in the management or administration of offenders on a day to day basis, e.g. a prison, a probation office, a court. NOTE: The following special Agency Location cases exist: - Agy Loc Id = TRN - Represents a Pending Transfer status typically used following an Offender Transfer Out operation and awaiting a Transfer In at the destination establishment; Agy Loc Id = OUT - Represents an Outside status typically associated with offenders that are the subject of an out of jurisdiction transfer or personal property that has been disposed to relatives, for example.",
+  "contacts": {
+    "author": "dominic.messenger@digital.justice.gov.uk",
+    "team": "https://dsdmoj.atlassian.net/wiki/spaces/DPR/overview?homepageId=4015489077",
+    "slack": "#ask_dpr"
+  },
+  "database": {
+    "type": "oracle",
+    "version": "12",
+    "name": "CNOMT3"
+  },
+  "fields": [
+    {
+      "name": "AGY_LOC_ID",
+      "type": "long",
+      "nullable": false,
+      "description": "The location residing within an agency. PK",
+      "glossary-index": "location:id",
+      "key": "primary"
+    },
+    {
+      "name": "DESCRIPTION",
+      "type": "string",
+      "nullable": true,
+      "description": "Description of the agency location."
+    },
+    {
+      "name": "AGENCY_LOCATION_TYPE",
+      "type": "string",
+      "nullable": true,
+      "glossary-index": "location:id",
+      "description": "Reference Code [ AGY_LOC_TYPE ]",
+      "key": "foreign"
```

Comment:
This is a reference code, which is probably represented as an Enum. Needs discussion with BA on how reference codes will be represented.
Not a foreign key

Reply:
Enum is not part of this ticket and all reference data not part of SOW1. confirmed by Zahoor